.pre-commit-config.yaml

```
@@ -20,5 +20,13 @@ repos:
 - repo: https://github.com/pre-commit/pre-commit-hooks
   rev: v4.3.0
```

```suggestion
  rev: v4.4.0
```
Latest version update

Also, I'd recommend running ``pre-commit autoupdate`` just in case any other hook is outdated